src/system/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { forwardRef, Ref } from 'react';
+import { Spinner as ThemeSpinner, SpinnerProps, ThemeUIStyleObject } from 'theme-ui';
+import classNames from 'classnames';
+
+export interface ThemeSpinnerProps extends SpinnerProps {
+	sx?: ThemeUIStyleObject;
+	className?: string;
+	color?: string;
+	strokeWidth?: number;
+}
+
+export const Spinner = forwardRef< SVGSVGElement, ThemeSpinnerProps >(
+	(
+		{ sx, color = 'icon.helper', strokeWidth = 2, className, ...props }: ThemeSpinnerProps,
+		ref: Ref< SVGSVGElement >
+	) => (
+		<ThemeSpinner
+			as={ 'svg' }
```

Comment:
generally for string literals in jsx you use HTML attribute syntax
```suggestion
			as="svg"
```